src/trainer.py

```diff
@@ -96,22 +102,21 @@ def build_trainer(params, model, run_dir, metric_should_decrease=True):
                               'verbose': True})
         log.info('\tUsing ReduceLROnPlateau scheduler!')
 
-    train_params = Params(
-        {
-            'cuda_device': params['cuda'],
-            'patience': params['patience'],
-            'grad_norm': params['max_grad_norm'],
-            'val_interval': params['val_interval'],
-            'max_vals': params['max_vals'],
-            'lr_decay': .99,
```

Comment:
This *really* shouldn't be set here. Worth looking into. That said, we can merge this before dealing with it.